site/docs/pages/wallet/types.mdx

```diff
@@ -13,10 +13,27 @@ type ConnectAccountReact = {
 };
 ```
 
-## `isValidAAEntrypointOptions`
+## `IsValidAAEntrypointOptions`
 
 ```ts
-export type isValidAAEntrypointOptions = {
+export type IsValidAAEntrypointOptions = {
   entrypoint: string;
 };
 ```
+
+## `IsWalletASmartWalletOptions`
+
+```ts
+export type IsWalletASmartWalletOptions = {
+  client: PublicClient;
+  userOp: UserOperation<'v0.6'>;
+};
+```
+
+## `IsWalletASmartWalletResponse`
+
+```ts
+export type IsWalletASmartWalletResponse =
+  | { isValid: true }
```

Comment:
I feel like this should say `isSmartWallet: boolean`, @cpcramer what do you think?

Reply:
I like `isSmartWalletResponse`, it's cleaner. If we use this then we should update the function to be `isSmartWallet`. 

 But I think we should use `IsWalletASmartWalletResponse` since the functions name is `isWalletASmartWallet`